### DISCUSSION TITLE
2023.2.0b1

### DISCUSSION THREAD

**Discussion by Djelibeybi** [Djelibeybi/ha-lifx-beta#44] [Announcements]
## What's Changed

* Remove the device-wide lock and sensor coordinator by @Djelibeybi in https://github.com/Djelibeybi/ha-lifx-beta/pull/43

**Full Changelog**: https://github.com/Djelibeybi/ha-lifx-beta/compare/2023.1.0b6...2023.2.0b1

<hr /><em>This discussion was created from the release <a href='https://github.com/Djelibeybi/ha-lifx-beta/releases/tag/2023.2.0b1'>2023.2.0b1</a>.</em>

**Comment by Madj42** [comment]
Found this version by accident because I've been looking into the LIFX connectivity issues I've had lately with HA.  My devices will randomly disconnect in HA but work with the Android app.  Installed this version via HACS with one of my devices marked unavailable in HA.  I used to see errors in the logs immediately at startup and with this I do not see those.  I noticed that the device was still offline after the HA restart but came online within the next minute after startup completed.  I will continue to monitor and report back any findings.  Thank you!

**Reply by Madj42** [reply]
Running HA 2023.2.2 and I'm in the monthly beta program as well.

**Reply by Djelibeybi** [reply]
Thanks for the feedback. The latest version isn't quite as aggressive about keeping bulbs online as the last one was (it went a bit over the top) but I am experiencing a similar outcome: more of my fleet are staying online and if they do go offline, they come back quicker.
